Doc/c-api/long.rst

```diff
@@ -571,6 +571,42 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    .. versionadded:: 3.14
 
 
+.. c:function:: int PyLong_IsPositive(PyObject *obj)
+
+   Check if the integer object *obj* is positive.
+
+   On success, return 1 if *obj* is positive, and 0 otherwise.
+
+   On failure, return -1 with an exception set.  This function always succeeds
+   if *obj* is a :c:type:`PyLongObject` or its subtype.
```

Comment:
```suggestion
   On success, return ``1`` if *obj* is positive, and ``0`` otherwise.

   On failure, return ``-1`` with an exception set.  This function always succeeds
   if *obj* is a :c:type:`PyLongObject` or its subtype.
```

Same suggestion for the 2 functions below.